contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1814,65 +1885,125 @@ class tsqlBuilder : public tsqlCommonMutator
 		}
 
 		// Declare table type statement might need rewriting in column definition list.
-		if (ctx->declare_statement() && ctx->declare_statement()->table_type_definition())
+		else if (ctx->declare_statement() && ctx->declare_statement()->table_type_definition())
```

Comment:
I have a bit concern about the else in here
Since I'm not 100% sure there'll be one statement match two if condition in below.

Reply:
There are many other examples of such constructs in the code already, and these work fine. For example:
- https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_X_DEV/contrib/babelfishpg_tsql/src/tsqlIface.cpp#L1648
- https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_X_DEV/contrib/babelfishpg_tsql/src/tsqlIface.cpp#L1766
- https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_X_DEV/contrib/babelfishpg_tsql/src/tsqlIface.cpp#L2550
- https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_X_DEV/contrib/babelfishpg_tsql/src/tsqlIface.cpp#L2995
- (and more)